wrappers/rust/icicle-core/src/ntt/tests.rs

```diff
@@ -27,6 +30,13 @@ where
     }
 }
 
+pub fn rel_domain<F: FieldImpl>(ctx: &DeviceContext)
```

Comment:
if this is the exposed API I would rename to release_domain

Reply:
Its a test function so I think the naming is fine but it doesn't need to be public since it is only being used in https://github.com/ingonyama-zk/icicle/pull/465/files#diff-96ad6a0ca42317faaf6524296cdaa92fcc3c634d36e2882c2b322d9b7287fd84R378